addons/point_of_sale/static/src/js/screens.js

```diff
@@ -2250,11 +2250,59 @@ define_action_button({
     },
 });
 
+
+/*--------------------------------------*\
+ |         THE LOGIN SCREEN           |
+\*======================================*/
+
+// The login screen enables employees to log in to the PoS
+// at startup or after it was locked, with either barcode, pin, or both.
+
+var LoginScreenWidget = ScreenWidget.extend({
+    template: 'LoginScreenWidget',
+
+    /**
+     * @override
+     */
+    show: function() {
+        var self = this;
+        this.$('.select-employee').click(function() {
+            self.gui.select_employee({
+                'security': true,
+                'current_employee': self.pos.get_cashier(),
+                'title':_t('Change Cashier'),})
+            .then(function(employee){
+                self.pos.set_cashier(employee);
+                $('.username').text(employee.name);
```

Comment:
Can you try to instead call `renderElement` of the UsernameWidget? This way you avoid the global CSS selector and when people write apps that modify that widget this will keep working.